runtime/bcutil/ROMClassBuilder.cpp

```diff
@@ -620,8 +618,11 @@ ROMClassBuilder::prepareAndLaydown( BufferManager *bufferManager, ClassFileParse
 		bool romClassIsShared = (j9shr_Query_IsAddressInCache(_javaVM, romClass, romClass->romSize) ? true : false);
 
 		if (compareROMClassForEquality((U_8*)romClass, romClassIsShared, &romClassWriter,
```

Comment:
How far could I go with this within the functions that I am editing? I've noticed quite a bit of it, but didn't want to have too many unrelated changes. 

A big thing for me is wanting to migrate to C++ casts (although not mentioned in the OMR coding standards doc, which I also believe is a mistake).

Reply:
just fixing the lines touched should be sufficient. larger updates can be pursued in separate PRs.

C++ casts are better than C casts. there might be a way to automate the conversion everywhere.

Reply:
Ok, sure.

Automating the cast conversion would be a challenge since C-style casts don't carry the appropriate information to automatically map on to the C++ casts. I think it would mostly be manual work.